client/common/common.c

```diff
@@ -188,6 +192,7 @@ usage(FILE *out, const char *name)
           " -e, --vim-esc-exits   exit bemenu when pressing escape in normal mode for vim bindings\n"
           " --accept-single       immediately return if there is only one item.\n"
           " --ifne                only display menu if there are items.\n"
+          " --multi-instance-mode do not force bemenu to run in only a single instance.\n"
```

Comment:
I think this flag can be omitted all together. Can't think of use case where this is needed.